src/components/EntityInfo.vue

```diff
@@ -6,7 +6,7 @@
     <v-container class="pt-0 pb-0">
       <v-row
         no-gutters
-        class="pt-3 pb-4"
+        class="pt-7 pb-7"
```

Comment:
If you wanted, you could use "py-7" to set both top and bottom.

Reply:
Oh yeah, totally forgot about that 👍